packages/ng/listing/listing.component.html

```diff
@@ -0,0 +1,11 @@
+@if (!ordered()) {
+<ul class="listing" [class.mod-icons]="icons()" [class.mod-checklist]="checklist()" [class]="paletteClass">
+	<ng-container *ngTemplateOutlet="content"></ng-container>
+</ul>
+} @else {
+<ol class="listing" [class.mod-icons]="icons()" [class.mod-checklist]="checklist()" [class]="paletteClass">
+	<ng-container *ngTemplateOutlet="content"></ng-container>
+</ol>
+}
```

Comment:
I'd prefer that inverted, mostly because checking for the inverse of a boolean when you have an else seems redundant.